lib/views/index.erb

```diff
@@ -0,0 +1,33 @@
+
+<body>
+
+  <div class="col-md-4 session-buttons">
+    <button><a href="/users/new">Sign Up</a></button>
+    <button><a href="/sessions/new">Sign In</a></button>
+  </div>
+
+  <div class="col-md-4">
+    <% if session[:user_id] %>
```

Comment:
Don't access session entries from your presentation layer.  You have a `current_user` helper method for this purpose!